pybamm/expression_tree/array.py

```diff
@@ -101,3 +101,16 @@ def new_copy(self):
     def _base_evaluate(self, t=None, y=None, y_dot=None, inputs=None):
         """ See :meth:`pybamm.Symbol._base_evaluate()`. """
         return self._entries
+
+
+def linspace(start, stop, num=50, **kwargs):
+    "Creates a linearly spaced array"
```

Comment:
why pick out `num` from the `kwargs`? Also, link to `np.linspace` docs for the kwargs

Reply:
did this so you can do `pybamm.linspace(0,1,100)` without needing to do `num=100` as I think it's a pretty common use case and you can just do `np.linspace(0,1,100)` directly. that didn't work unless I picked out `num` but maybe there is a better/more common solution?